featurebyte/service/deploy.py

```diff
@@ -228,10 +228,9 @@ async def _revert_changes(
             return_document=False,
         )
 
-    async def update_feature_list(
+    async def _update_feature_list(
```

Comment:
Make this private as the deployment should be handled through deployment object rather than feature list.